trunk/includes/pardot-api-class.php

```diff
@@ -60,74 +62,39 @@ class Pardot_API {
 	private static $LOGIN_URL_PATH_TEMPLATE = '/login/version/%%VERSION%%';
 
 	/**
-	 * @var string A user entered email address that is expected to have a valid Pardot account.
+	 * @var string A user-entered client ID.
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $email;
+	var $client_id;
 
 	/**
-	 * @var string A user-entered password for the valid Pardot account indentified by $this->email.
+	 * @var string A user-entered client secret.
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $password;
+	var $client_secret;
 
 	/**
-	 * @var string A user-entered but Pardot-supplied key the user accesses from their Pardot account.
+	 * @var string A user-entered business unit ID.
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $user_key;
-
-    /**
-     * @var string The type of authentication, either Pardot or SSO.
-     *
-     * @since 1.5.0
-     */
-    var $auth_type;
-
-    /**
-     * @var string A user-entered client ID.
-     *
-     * @since 1.5.0
-     */
-    var $client_id;
-
-    /**
-     * @var string A user-entered client secret.
-     *
-     * @since 1.5.0
-     */
-    var $client_secret;
-
-    /**
-     * @var string A user-entered business unit ID.
-     *
-     * @since 1.5.0
-     */
-    var $business_unit_id;
+	var $business_unit_id;
 
 	/**
-	 * @var string A key returned on authentication by Pardot or SSO
+	 * @var string A key returned on authentication by SSO
 	 *
 	 * @since 1.5.0
 	 */
 	var $api_key = false;
 
-    /**
-     * @var string A refresh token returned on authentication by SSO; used to get new api_key
-     *
-     * @since 1.5.0
-     */
-    var $refresh_token = false;
-
 	/**
-	 * @var boolean Used to flag the API for retry in case the api_key has expired.
+	 * @var string A refresh token returned on authentication by SSO; used to get new api_key
 	 *
-	 * @since 1.0.0
+	 * @since 1.5.0
 	 */
-	var $api_key_maybe_invalidated = false;
```

Comment:
I don't believe this is necessary anymore.  When our access_token (aka api_key) expires, we will refresh the token and make the call again automatically.